cvat_proxy/conf.d/cvat.conf.template

```diff
@@ -0,0 +1,37 @@
+server {
```

Comment:
Let's rename the directory ``cvat_proxy`` to ``cvat-proxy`` just to be consistent

Reply:
Let's merge as is but it should be consistent. Agree.